media/gpu/test/BUILD.gn

```diff
@@ -41,6 +41,8 @@ if (enable_av1_decoder && (use_v4l2_codec || use_vaapi)) {
     ]
   }
 
+  # TODO(b/386420709): These fail to link with Cobalt config.
+  if (!is_cobalt) {
```

Comment:
Why is this needed if we're not building `all`?

Reply:
This should not have been part of this PR. It belongs in https://github.com/youtube/cobalt/pull/4464 where it got picked up by the test target calculation script.

I'll remove it from this change.